app/mailers/devise_custom_mailer.rb

```diff
@@ -13,7 +13,7 @@ def confirmation_instructions(record, token, opts = {})
     opts[:subject] = I18n.t('mailers.confirmation_instructions.subject')
     @reply_subject = 'Pie for Providers: question after signup'
```

Comment:
does this need I18n.t ?

Reply:
No, it's the subject **to us** if they click that email to reply.